server/src/main/java/org/kaazing/gateway/server/Launcher.java

```diff
@@ -45,18 +49,19 @@ public Launcher(GatewayObserver gatewayListener) {
         this.gatewayListener = gatewayListener;
     }
 
-    public void init(GatewayContext context) throws Exception {
+    public void init(GatewayContext context) throws URISyntaxException , IOException  {
```

Comment:
I don't think I want these specific exceptions in the contract. May be LauncherException a runtime exception with a specific cause